deps/build.jl

```diff
@@ -2,14 +2,58 @@
 import JSON
 using Compat
 
+# remove deps.jl if it exists, in case build.jl fails
+isfile("deps.jl") && rm("deps.jl")
+
+if isdir(abspath(dirname(@__FILE__), "..", "..", "Conda"))
+    Conda_installed = true
+    import Conda
+    Conda.add("ipython-notebook=3.1")
+else
+    Conda_installed = false
+end
+
 # print to stderr, since that is where Pkg prints its messages
 eprintln(x...) = println(STDERR, x...)
 
 # Make sure Python uses UTF-8 output for Unicode paths
 ENV["PYTHONIOENCODING"] = "UTF-8"
 
-include("jupyter.jl")
-const jupyter, jupyter_vers = find_jupyter()
+if Conda_installed
+    condapath=dirname(Conda.conda)
```

Comment:
This would need the `Conda.bindir(package)` to be implemented first I think. I think the easier way to do this would be to ask callers of this function to do the executable <=> package linking by themselves.